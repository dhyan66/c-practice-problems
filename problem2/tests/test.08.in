10 7
2
1
2
4
5
4
4
