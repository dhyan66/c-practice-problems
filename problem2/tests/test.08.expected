Odd: 1 4 5
